students/mxwllndrsn/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,53 @@
+#uw python 210
+#lesson 03
+#max anderson
+
+#slicing lab
+
+def switch_firstlast(seq):
+
+    return seq[-1:] + seq[1:len(seq)-1] + seq[0:1]
```

Comment:
Delete extra line in this function as well as following ones.

Reply:
not sure what you mean